packages/server/lib/db/migrations/20230326083713_create_cache.cjs

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex, _) {
+    return knex.schema.withSchema('nango').createTable('_nango_cache', function (table) {
+        table.string('key').notNullable();
```

Comment:
Don't you think it's better to use this table as a cache? The approach you propose means that every time we need to store something as a cache, we'll have to create a new table and add a new column for every new key in the Oauth_session. However, since Postgres supports JSON, it could provide us with a better opportunity for a flexible and scalable cache type.

Reply:
If we see this as a typical cache, I agree that it's fine to have schemaless storage. 

But this also can just be seen as a typical session object. Because we already have a strict SQL schema for web sessions (table `_nango_sessions`), I would do it symmetrically and store OAuth sessions with a schema as well, even if it means doing a couple migrations later on. 

When we see a use-case for a general-purpose cache, then we can always add Redis then. 

But let me know if you still disagree.